pkg/services/store/object/dummy/dummy_server.go

```diff
@@ -208,24 +205,18 @@ func (i dummyObjectServer) update(ctx context.Context, r *object.WriteObjectRequ
 }
 
 func (i dummyObjectServer) insert(ctx context.Context, r *object.WriteObjectRequest, namespace string) (*object.WriteObjectResponse, error) {
-	modifier := object.UserFromContext(ctx)
+	modifier := object.GetUserIDString(object.UserFromContext(ctx))
 	rawObj := &object.RawObject{
-		UID:     r.UID,
-		Kind:    r.Kind,
-		Updated: time.Now().Unix(),
-		Created: time.Now().Unix(),
-		CreatedBy: &object.UserInfo{
-			Id:    modifier.UserID,
-			Login: modifier.Login,
-		},
-		UpdatedBy: &object.UserInfo{
-			Id:    modifier.UserID,
-			Login: modifier.Login,
-		},
-		Size:    int64(len(r.Body)),
-		ETag:    createContentsHash(r.Body),
-		Body:    r.Body,
-		Version: fmt.Sprintf("%d", 1),
+		UID:       r.UID,
+		Kind:      r.Kind,
+		Updated:   time.Now().Unix(),
+		Created:   time.Now().Unix(),
+		CreatedBy: modifier,
```

Comment:
thought of a couple questions related to syncing objects from an external store:
* do we need to provide a way to set the timestamps and user info in `WriteObjectRequest` for objects that were created in the past?
* do we need to track modifications by people that will never use the grafana instance? i'm thinking of contributors to a public repo of dashboard examples or something

Reply:
> do we need to provide a way to set the timestamps 

TBD -- I a skeptical about making that easy to overwrite externally, but I do see the value.  

I think we can avoid it until we have a real need :)